docs/related/async-raml-datatypes.mdx

```diff
@@ -0,0 +1,324 @@
+---
+id: async_raml_datatypes
+title: Async + RAML Data Types in AMF
+---
+
+:::info Technical Document
+This is a technical document, this information is intended for experienced users
+:::
+
+Within the Async specification it is possible to use raml data types to define a payload, taking into account that the schema format is specified accordingly.
+Both specifications have their own distinct referencing and modularization mechanisms, and it is not clear where to draw the line to determine where one specification stops, and the other one starts.
+This document will describe how to use RAML types in Async when using AMF.
+
+
+## Referencing external RAML content using ‘$ref’
+In order to support referencing raml content in external files, AMF allows the use of the `$ref` mechanism present in Async but with clear restrictions.
```

Comment:
It's unclear what the "clear restrictions" are. Will these be delineated further down in this document? If so, the reader should be told this. If not, perhaps we should include these restrictions here?

Or are the restrictions mentioned in the next sentence? 

This is a little confusing I think.

Reply:
You're totally right, I've rewritten this part.